### DISCUSSION TITLE
Keep facetOrdering if custom sortBy passed to connectRefinementList

### DISCUSSION THREAD

**Discussion by jep-a** [algolia/instantsearch#4969] [Ideas]
Hello, I have a use case where I don't want the default sort to include `isRefined` when using the refinement list connector, so I pass a value of `["count:desc", "name:asc"]` to `sortBy`, but I still want any facet display ordering to override my sort if set.

Related line: [`src/connectors/refinement-list/connectRefinementList.ts#L388`](https://github.com/algolia/instantsearch.js/blob/master/src/connectors/refinement-list/connectRefinementList.ts#L388)

I realized that have to leave the default sort to keep `facetOrdering`. It'd be very useful to be able to customize both the default sort and still have the option to keep `facetOrdering` enabled.

**Comment by Haroenv** [comment]
We saw this possibly being confusing, but in the end indeed went with "only facet ordering, not mixing between sortBy & facetOrdering", as both options could be considered confusing. What you can do as a workaround is sort in `transformItems`, which happens *after* sortBy, by using transformItems:

```js
refinementList({
  sortBy: undefined,
  transformItems(items) { return items.sort((a, b) => a.isRefined < b.isRefined) }
})
```

example: https://codesandbox.io/s/kind-http-381ut?file=/src/app.js
